src/components/Dashboard/FilingHistoryList/filings/StaffFiling.vue

```diff
@@ -0,0 +1,94 @@
+<template>
```

Comment:
Special case for all staff filings (of different types) instead of having all separate components.